src/DetailsView/details-view-initializer.ts

```diff
@@ -275,6 +277,12 @@ if (isNaN(tabId) === false) {
                     urlParser,
                     dateProvider: DateProvider.getDate,
                     settingsProvider: SettingsProviderImpl,
+                    environmentInfoProvider: new EnvironmentInfoProvider(
```

Comment:
nit: Is it better to create this object in a new line and use that variable/object here?